pyglmnet/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .pyglmnet import GLM, set_log_level
 from .utils import softmax, label_binarizer, log_likelihood
 from .datasets import fetch_tikhonov_data
-__version__ = '1.0.0'
+__version__ = '1.0.1'
```

Comment:
in semver.org the idea is to version as `MAJOR.MINOR.PATCH`. my thought was that until the next PyPI release (which should be called `2.0.0`), we can categorize our PRs into `MINOR` or `PATCH` and accordingly increment the version number. do you have a suggestion?